lib/transcriptomics/transcriptomics_setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -Euo pipefail
+
+LOGFILE=$PWD/tmp/progress.txt
+
+# This script runs after the container is composed.
+
+echo ">> waiting for transcriptomics to start"
+transcriptomics=$(docker ps --format "{{.Names}}" | grep transcriptomics_1)
+while [ $? -ne 0 ]
+do
+  echo "..."
+  sleep 1
+  transcriptomics=$(docker ps --format "{{.Names}}" | grep transcriptomics_1)
+done
+sleep 5
```

Comment:
You probably want to actually do some setup here after the container is running. If we're going to use service tokens, this module will need to create its own service store.
```suggestion
sleep 5

python settings.py
source env.sh
bash $PWD/create_service_store.sh "transcriptomics"

```